samples/lib/widgets/select_group.dart

```diff
@@ -73,7 +73,7 @@ class CheckboxFormState extends State<CheckboxForm> {
           mainAxisAlignment: MainAxisAlignment.center,
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
-            FSelectGroup(
+            FSelectGroup<Language>(
```

Comment:
Do we need to type this? Sad if we do :(

Reply:
Yes sadly :( Dart's type inference fails here.

Reply:
:o I don't remember it throwing any compile time errors. Could you update the code sample in the docs, I don't think I have explicit generics included.